docs/sdk/serverless-signing/overview.md

```diff
@@ -5,39 +5,47 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 :::info
 **STATE OF THE NETWORK**
 
-Using Lit Actions in production IS now supported on the [Datil Mainnet](../../network/networks/mainnet) and [Datil Testnet](../../network/networks/testnet). Check out the [docs on migration](../../network/migration-guide) to learn how you can start building on Datil today. 
+Using Lit Actions in production IS now supported on the [Datil Mainnet](../../network/networks/mainnet). Check out the [docs on migration](../../network/migration-guide) to learn how you can start building on Datil today. 
 
 :::
```

Comment:
I think this should be removed. Makes it seem like LA weren't supported on Datil at one point, and LA are supported on every available Lit network now anyways